nexus_constructor/InstrumentView.py

```diff
@@ -0,0 +1,157 @@
+from PySide2.QtWidgets import QWidget, QVBoxLayout
+from PySide2.Qt3DExtras import Qt3DExtras
+from PySide2.Qt3DCore import Qt3DCore
+from PySide2.QtCore import QPropertyAnimation
+from PySide2.QtGui import QVector3D, QColor, QMatrix4x4
+from nexus_constructor.NeutronAnimationController import NeutronAnimationController
+
+
+class InstrumentView(QWidget):
```

Comment:
Perhaps just add a docstring here describing the overall purpose of the class, and same with `NeutronAnimationController`.